slippi-stats.js

```diff
@@ -71,250 +76,311 @@ var final_player_name = user_player
 
 console.log(`${files.length} replays found.`)
 
-for (i = 0; i < files.length; i++) {
```

Comment:
Note that the diff itself will look huge, but it's actually very small. I made no changes to the logic, but because I had to allocate the task pool before I did the file iteration, git thinks I changed everything.